packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -116,18 +123,22 @@ function UncontrolledInnerBlocks( props ) {
 
 	// This component needs to always be synchronous as it's the one changing
 	// the async mode depending on the block selection.
```

Comment:
I feel this commit is misplaced now no?

Reply:
Yes, it actually doesn't look relevant even in trunk. I think this was from before we created BlockListItems and used the async provider directly here.

Reply:
yep